pocs/tests/test_observatory.py

```diff
@@ -25,7 +26,11 @@ def simulator():
 @pytest.fixture
 def observatory(config, simulator):
     """Return a valid Observatory instance with a specific config."""
-    obs = Observatory(config=config, simulator=simulator, ignore_local_config=True)
+    cameras = create_cameras_from_config(config)
```

Comment:
local=False 

Or some such to prevent the pocs_local.yaml from being picked up?

Reply:
Doesn't the `config` fixture handle that?

Reply:
Oh, it may do so. NM.